Cargo.toml

```diff
@@ -5,11 +5,12 @@ edition = "2021"
 
 [dependencies]
 datafusion = "13.0"
+datafusion-sql = "13.0"
```

Comment:
We should not need to add this since we already depend on `datafusion`, which re-exports all the other crates.